textalyze.js

```diff
@@ -55,14 +57,21 @@ module.exports = { itemCounts, arrayFrom, sanitize };
 // running the app
 //
 if (require.main === module) {
-  const string = 'Hello World';
+  const path = './sample_data/tale-of-two-cities.txt';
+
+  fs.readFile(path, (err, data) => {
+    if (err) {
+      console.error(err);
+      return;
```

Comment:
This top-level code is really the "user interface", so I'd call `process.exit(1)` here.  We want the program to end with an error [exit code](https://shapeshed.com/unix-exit-codes/) (anything but `0` is considered an error).